### PR TITLE
Hard code redis host for 12 factor apps

### PR DESCRIPTION
**Pull request by tijmenb**
This is a temporary fix for the problem that some sidekiq-monitoring apps can't access the `REDIS_HOST` environment variable from "their" application. 

A more sustainable fix is forthcoming, but this will make sure we have sidekiq monitoring for dfid, publishing-api, travel-advice-publisher and whitehall.

### CODE REVIEW

File: dfid-transition/config.ru [hunk 1]
```diff
@@ -2,7 +2,7 @@ require 'sidekiq'
 
 Sidekiq.configure_client do |config|
   config.redis = {
-    host: ENV["REDIS_HOST"] || "127.0.0.1",
+    host: "redis-1.backend",
```
Comment (andrewgarner) [inline dfid-transition/config.ru]: Could we use `redis-1.backend` as a default instead of `127.0.0.1`?

Comment (tijmenb) [inline dfid-transition/config.ru]: I think localhost is used for local development. Perhaps it's clearer in the short term if we hard code it like this?

File: Procfile [hunk 2]
```diff
@@ -1,10 +1,10 @@
-publisher: cd publisher && LC_ALL=en_GB.UTF-8 exec govuk_setenv publisher bundle exec rackup -p 3079
```
Comment (andrewgarner) [inline Procfile]: Can we leave this in or will it fail if `/etc/govuk/publisher/env.d` is missing on the box?

Comment (tijmenb) [inline Procfile]: Yeah, it will fail.

Comment (andrewgarner) [inline Procfile]: Fair enough!

**Comment by andrewgarner** [comment]
👍

**Pull request merged**